static/js/EditFilenameForm.js

```diff
@@ -0,0 +1,53 @@
+define('editFilenameForm', [ 'jquery', 'jquery-ui', 'underscore', 'RemoteStateChangeMonitor', 'tpl!../templates/admin/editFilenameForm', 
```

Comment:
This file should be located in the same directory as other admin js classes, so static/js/admin/src/